packages/cozy-client/src/associations/HasManyAssociation.js

```diff
@@ -0,0 +1,62 @@
+export default class HasManyAssociation {
+  constructor(target, name, { get, query, mutate }) {
+    this.target = target
+    this.name = name
+    this.get = get
+    this.query = query
+    this.mutate = mutate
+  }
+
+  get data() {
```

Comment:
With getter it seems it is not possible to target IE8.

Reply:
And I don't give a shit ;)